pkg/apis/componentconfig/validation/validation_pluginargs.go

```diff
@@ -142,3 +149,53 @@ func validatePodLifeTimeStates(states []string) error {
 
 	return nil
 }
+
+func ValidateHighNodeUtilizationArgs(args *componentconfig.HighNodeUtilizationArgs) error {
+	return errorsAggregate(
+		validateThresholds(args.Thresholds),
+	)
```

Comment:
nit: no need to aggregate if it's a single error
```suggestion
	return validateThresholds(args.Thresholds)
```